### PR TITLE
Add support for native templates

### PR DESCRIPTION
**Pull request by spadgett**
https://jira.coreos.com/browse/CONSOLE-889

@sspeiche @alimobrem @jwforres fyi

Screen Shot 2019-04-23 at 4 00 29 PM

Screen Shot 2019-04-26 at 12 55 33 PM

### CODE REVIEW

**Comment by spadgett** [comment]
Added support for template instances. This should be ready for review.

/assign @rhamilto 

Screen Shot 2019-04-26 at 3 52 13 PM

Screen Shot 2019-04-26 at 3 50 31 PM

**Comment by spadgett** [comment]
Screen Shot 2019-04-26 at 4 02 35 PM

File: frontend/public/components/instantiate-template.tsx [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+
+import * as React from 'react';
+import { connect } from 'react-redux';
+import * as _ from 'lodash-es';
+import { Helmet } from 'react-helmet';
+import * as classNames from 'classnames';
+
+import { ANNOTATIONS } from '../const';
+import { getImageForIconClass, getTemplateIcon, normalizeIconClass } from './catalog/catalog-item-icon';
+import { LoadingBox, LoadError } from './utils/status-box';
```
Comment (rhamilto) [inline frontend/public/components/instantiate-template.tsx]: Could be combined with line 12.

**Comment by spadgett** [comment]
Thanks @rhamilto, updated

**Pull request merged**